### ISSUE TITLE
auv viewer won't load

### ISSUE THREAD

**Issue by lbesnard** [aodn/auv#28]
output from firebug
```
Blocked loading mixed active content "http://fonts.googleapis.com/css?family=Lobster"
```

if this link is changed with https, the font is loaded. But why using a special font ?

and links such as
https://auv.aodn.org.au/auv/proxy?url=https%3A%2F%2Fgeoserver-123.aodn.org.au%2Fgeoserver%2Fwms%3FLAYERS%3Dimos%253Aauv_tracks_vw%26STYLES%3D%26SRS%3DEPSG%253A4326%26FORMAT%3Dimage%252Fpng%26TRANSPARENT%3DTRUE%26TILED%3Dtrue%26SERVICE%3DWMS%26VERSION%3D1.1.1%26REQUEST%3DGetFeatureInfo%26EXCEPTIONS%3Dapplication%252Fvnd.ogc.se_xml%26CQL_FILTER%3Dsite_code%2520like%2520%27r20101120_222627_site2guz_14_densegrids%27%26BBOX%3D150.38793%252C-35.569678%252C150.396513%252C-35.562812%26INFO_FORMAT%3Dtext%252Fhtml%26QUERY_LAYERS%3Dimos%253Aauv_tracks_vw%26FEATURE_COUNT%3D100%26WIDTH%3D400%26HEIGHT%3D320%26X%3D62%26Y%3D125

fail, however all links in the page ( claiming the 500 error) actually work

same here
https://auv.aodn.org.au/auv/proxy?url=https%3A%2F%2Fgeoserver-123.aodn.org.au%2Fgeoserver%2Fwms%3FLAYERS%3Dimos%253Aauv_images_vw%26STYLES%3D%26SRS%3DEPSG%253A4326%26FORMAT%3Dimage%252Fpng%26TRANSPARENT%3DTRUE%26TILED%3Dtrue%26SERVICE%3DWMS%26VERSION%3D1.1.1%26REQUEST%3DGetFeatureInfo%26EXCEPTIONS%3Dapplication%252Fvnd.ogc.se_xml%26CQL_FILTER%3Dsite_code%2520like%2520%27r20101121_062424_site1sz_16_densegrids3%27%26BBOX%3D150.437054%252C-35.533392%252C150.441346%252C-35.529958%26INFO_FORMAT%3Dtext%252Fhtml%26QUERY_LAYERS%3Dimos%253Aauv_images_vw%26FEATURE_COUNT%3D100%26WIDTH%3D400%26HEIGHT%3D320%26X%3D219%26Y%3D63

but 
https://geoserver-123.aodn.org.au/geoserver/wms?LAYERS=imos%3Aauv_images_vw&STYLES=&SRS=EPSG%3A4326&FORMAT=image%2Fpng&TRANSPARENT=TRUE&TILED=true&SERVICE=WMS&VERSION=1.1.1&REQUEST=GetFeatureInfo&EXCEPTIONS=application%2Fvnd.ogc.se_xml&CQL_FILTER=site_code%20like%20%27r20101121_062424_site1sz_16_densegrids3%27&BBOX=150.437054%2C-35.533392%2C150.441346%2C-35.529958&INFO_FORMAT=text%2Fhtml&QUERY_LAYERS=imos%3Aauv_images_vw&FEATURE_COUNT=100&WIDTH=400&HEIGHT=320&X=219&Y=63

works

However the links are HTTPS and should be HTTP I reckon
__UPDATE__
page is greyed instead of being white before ( could be an error somewhere)

**Comment by lbesnard** [comment]
@pmbohm

**Comment by pmbohm** [comment]
The site looks great from home. 
The site is grey now! You were looking at it before!
I do see the console error (but it works) and it seems that will go away by using HTTPS: https://fonts.googleapis.com/css?family=Lobster

**Comment by danfruehauf** [comment]
@lbesnard You tagged again the wrong person.

**Comment by dnahodil** [comment]
@jkburges

**Comment by dnahodil** [comment]
Everyone gets tagged :tada:

**Comment by pmbohm** [comment]
fixed by 31ef6230e81b67447bbdfe59d3511367d50a05e7
